lib/request/rack.rb

```diff
@@ -136,6 +136,7 @@ def if_modified_since
     accessor(:host,         Key.new('SERVER_NAME'))
     accessor(:query_string, Key.new('QUERY_STRING'))
     accessor(:content_type, Key.new('CONTENT_TYPE'), nil)
+    accessor(:cookies,      Key.new('HTTP_COOKIE'), EMPTY_STRING)
```

Comment:
@snusnu Yeah I'd return `nil` in case this header is unset.

I also think we might want to name the method differently. `cookies` (at least for me) triggers the imagination of a collection of objects.

Maybe we want to name it `raw_cookie` ?

I wounder if we should simply merge `request` and `cookie`.